cms/jinja2/templates/components/streamfield/release_date_change_block.html

```diff
@@ -0,0 +1,6 @@
+<li class="ons-list__item">
+    <h3 class="ons-u-mb-no ons-u-fs-r--b">{{ _("Previous date") }}</h3>
+    <p>{{ value.previous_date|date("j F Y g:ia") }}</p>
```

Comment:
Suggestion: Should this format be defined globally, since I suspect all will be shown in a standard way

Reply:
was changed to `|date("DATETIME_FORMAT")`